### DISCUSSION TITLE
Bullseye, SimpleExec, NUKE. Alternatives to CAKE?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [Dotnet-Boxed/Templates#360] [Q&A]
I see this is growing in popularity https://pknopf.com/post/2019-03-10-you-dont-need-cake-anymore-the-way-to-build-dotnet-projects-going-forward/

In fact, IdentityServer seems to do it: https://github.com/IdentityServer/IdentityServer4/tree/master/src/IdentityServer4/build

Is this something we should consider?

**Comment by RehanSaeed** [comment]
Good question. I've been aware of Nuke for a while. I've already been using Cake without any problems, also switching is work, so I'm not itching to do it.

If you look at the IdentityServer script, it's a bit less expressive than Cake, just because it doesn't offer any of the little helper methods. I actually think the cake script is just a bit easier to read. Cake has a pretty active community (I've also contributed), so it's well maintained. The GitHub stars for the project are proof of this.

With that said, I can see why Nuke has certain advantages. I might even give it a try the next time I write a build script. It would be an interesting exercise to see what the script for this project would look like.
